modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]DefineAPISchemasWithinAnAPIGroup/CreateAPIApplicationSchema.testcase

```diff
@@ -0,0 +1,207 @@
+		property portal.acceptance = "true";
+		property test.liferay.virtual.instance = "false";
+		property test.run.type = "single";
+
+		task ("And Given a Schema created in the API Application") {
+			SchemaAPI.createAPISchema(
+				apiApplicationId = ${staticApplicationId},
+				mainObjectDefinitionERC = "L_API_APPLICATION",
+				name = "testSchema_1");
+		}
+
+		task ("When I click Add New Schema") {
+			APIBuilderUI.createAPISchema(
+				confirmCreation = "true",
+				name = "testSchema_2",
+				objectDefinitionName = "APIApplication");
+		}
+
+		task ("Then I'm able to create another schema for the API Application") {
+			APIBuilderUI.switchToRelatedEntryInEditApplication(
+				entity = "Schemas",
+				title = "My-app");
+
+			APIBuilderUI.viewItemsInOrder(itemList = "testSchema_1,testSchema_2");
+		}
+	}
+
+	@priority = 4
+	test CanSeeInfoOfCreatedSchema {
+		property portal.acceptance = "true";
+		property test.liferay.virtual.instance = "false";
+		property test.run.type = "single";
+
+		task ("When I create a schema with valid Name and Object") {
+			APIBuilderUI.createAPISchema(
+				confirmCreation = "true",
+				name = "testSchema",
+				objectDefinitionName = "APIApplication");
+		}
+
+		task ("Then I can see details info of the created schema") {
+			AssertTextEquals(
+				locator1 = "APIBuilder#ENTER_NAME",
+				value1 = "testSchema");
+
+			AssertTextEquals(
+				locator1 = "APIBuilder#SELECT_AN_OBJECT_DEFINITION",
+				objectDefinitionName = "APIApplication",
+				value1 = "APIApplication");
+		}
+	}
+
+	@priority = 5
+	test SchemaIsNotCreatedAfterClickingCancel {
+		property portal.acceptance = "true";
+		property test.liferay.virtual.instance = "false";
+		property test.run.type = "single";
+
+		task ("And Given I fill in valid Name and Object fields") {
+			APIBuilderUI.createAPISchema(
+				name = "testSchema",
+				objectDefinitionName = "APIApplication");
+		}
+
+		task ("When I click Cancel button") {
+			Click(locator1 = "Button#CANCEL");
+		}
+
+		task ("Then no new schema on the list of schemas") {
+			AssertElementNotPresent(
+				key_name = "testSchema",
+				locator1 = "ObjectAdmin#TABLE_LIST_TITLE");
+		}
+	}
+
+	@priority = 5
+	test SchemaNameMustBeUnique {
+		property portal.acceptance = "true";
+		property test.liferay.virtual.instance = "false";
+		property test.run.type = "single";
+
+		task ("And Given a Schema 'new' created in the API Application") {
+			SchemaAPI.createAPISchema(
+				apiApplicationId = ${staticApplicationId},
+				mainObjectDefinitionERC = "L_API_APPLICATION",
+				name = "new");
+		}
+
+		task ("When I try creating another schema with name 'new'") {
+			APIBuilderUI.createAPISchema(
+				confirmCreation = "true",
+				name = "new",
+				objectDefinitionName = "APIApplication");
+		}
+
+		task ("Then an error message 'There is an API schema with the same name in the API application' blocks schema from being created") {
+			Click(locator1 = "Button#CANCEL");
```

Comment:
If we click Cancel button, then we can not get the error message, please check and update

Reply:
oh, there is no need to click cancel I will remove it